components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -1,13 +1,133 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import dynamic from 'next/dynamic';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 import { formatCurrency } from '../../../lib/currency-utils';
+import { days } from '../../../lib/utils';
 
-import { Box } from '../../Grid';
+import Container from '../../Container';
+import { Box, Flex } from '../../Grid';
+import Loading from '../../Loading';
 import LoadingPlaceholder from '../../LoadingPlaceholder';
 import ProportionalAreaChart from '../../ProportionalAreaChart';
 import { P, Span } from '../../Text';
+const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
+
+const getChartOptions = (intl, startDate, endDate) => {
+  return {
+    chart: {
+      id: 'chart-transactions-overview',
+    },
+    legend: {
+      show: true,
+      horizontalAlign: 'left',
+    },
+    colors: ['#29CC75', '#F55882'],
+    grid: {
+      xaxis: { lines: { show: true } },
+      yaxis: { lines: { show: false } },
+    },
+    stroke: {
+      curve: 'straight',
+      width: 1.5,
+    },
+    dataLabels: {
+      enabled: false,
+    },
+
+    xaxis: {
+      categories: getCategories(intl, startDate, endDate),
+    },
+    yaxis: {
+      labels: {
+        formatter: function (value) {
+          return value < 1000 ? value : `${Math.round(value / 1000)}k`;
+        },
+      },
+    },
+  };
+};
+
+const getNumberOfDays = (startDate, endDate) => {
+  const startTimeOfStatistics = new Date(2015, 0, 1);
+  let numberOfDays;
+  if (startDate && endDate) {
+    numberOfDays = days(new Date(startDate), new Date(endDate));
+  } else if (startDate) {
+    numberOfDays = days(new Date(startDate));
+  } else if (endDate) {
+    numberOfDays = days(startTimeOfStatistics, new Date(endDate));
+  } else {
+    numberOfDays = days(startTimeOfStatistics);
+  }
+  return numberOfDays;
```

Comment:
We can probably simplify with something like

```suggestion
  return dayjs(startDate || host.createdAt).diff(dayjs(endDate), 'day');
```

Reply:
Actually, we already have, `days` function in `util.js` and I think we can get rid of this function and use that instead, which I've done. 😉